nupic/research/frameworks/dendrites/functional/apply_dendrites.py

```diff
@@ -86,22 +101,23 @@ def dendritic_gate_2d(y, dendrite_activations):
               (b, c, h, w) where the axes represent the batch, channel, height, and
               width dimensions respectively)
     :param dendrite_activations: the dendrite activation values (a torch tensor
-                                 with shape (b, c) where the axes represent the
-                                 batch and channel dimensions, respectively)
+                                 with shape (b, c, d) where the axes represent the
+                                 batch, channel, and dendrite dimensions, respectively)
```

Comment:
For consistent terminology, you should replace "dendrite" with "segment" here. You can rephrase so that "num segments" gets mentioned.

Reply:
Updated to use "number of units" and "number of segments".